### ISSUE TITLE
Multiboxing problem sometimes

### ISSUE THREAD

**Issue by sucidaldeathtar** [EasyFarm/EasyFarm#221]
I run a party using this on a private server and sometimes it will attack one target then stop.  I start it again then stops after next mob.  Im using windower and after a little it will crash the windower client as well.

Other times all 6 will work fine.  Is there something I can do to stop this?

**Comment by Mykezero** [comment]
Yeah, a few people have mentioned that multiboxing is kind of iffy. I'll see what I can do about this. Sorry for the late response >.<

As for Windower crashing, I've noticed that when EF is running, the CPU for the FFXI process shoots up a bit. I think this is caused by the program rapidly interacting with the game.  

Sadly, to fix that problem, it would require a massive redesign of the program to somehow limit the interactions per second to ensure that it does not overload the FFXI game process.
